src/PKSim.Presentation/Presenters/Individuals/ShowOntogenyDataPresenter.cs

```diff
@@ -94,9 +97,17 @@ private void updateChart()
          _simpleChartPresenter.Refresh();
       }
 
+
+      private IEnumerable<DataColumn> orderOntogenyColumns(IEnumerable<DataColumn> dataColumns)
+      {
+         //we need to inverse the order of columns so that standard deviation is last and ontogeny is first 
+         //this is the other way by constructions as the main curve is in fact the deviation and the related column is the mean
+         return dataColumns.Reverse();
```

Comment:
this is what needs to be done. This will fail if we have a more complex structure but accessing by name means that tI know how it's generated etc.. so this is just fine for now I believe